client/app/bundles/Statistics/components/Routes.jsx

```diff
@@ -6,7 +6,7 @@ import OfferCreatedPage from './OfferCreatedPage'
 import OfferApprovedPage from './OfferApprovedPage'
 import OrgaCreatedPage from './OrgaCreatedPage'
 import OrgaApprovedPage from './OrgaApprovedPage'
-import ProductivityGoalPage from './ProductivityGoalPage'
+import StatisticChartPage from './StatisticChartPage'
 
 export default class Statistics extends React.Component {
```

Comment:
Als ich gerade hier im Code angekommen war, ist mir aufgefallen, dass die aktuellen Statistiken (offer_approved, orga_approved aufgeteilt nach Stadt etc.) im Moment kaputt sind (JS-Fehler). Das sollte vor dem Deploy auf jeden Fall behoben werden.

Update: der Fehler ist, dass die methode navigateThroughSubmodels von rform nicht vorhanden ist und in der Tat habe ich die auch gerade in dem repository nicht gefunden. Muss da vllt. einfach noch etwas commited werden?!

in FilteringSelect.jsx
import { updateAction, navigateThroughSubmodels } from 'rform'